### ISSUE TITLE
Wrong Dutch Language translation in AG popup window

### ISSUE THREAD

**Issue by Kees1958** [AdguardTeam/AdguardBrowserExtension#1974]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x ] I am running the latest version
- [ x] I checked the documentation and found no answer
- [x ] I checked to make sure that this issue has not already been filed

### Issue Details

Current Dutch menu text is "Alle gebruikersregels herstellen voor deze webpagina" 

Two errors
1. Although the literal translation of "user rules" is "gebruikersregels" the menu item in Dutch is NOT called "gebruikersregels" but "eigen filter" (see attached picture)
2. The text says "herstellen" which in Dutch means "to repair" or "recover", while the action removes/deletes the user rules of that webpage. To remove/delete in Dutch is "verwijderen"

- **AdGuard version:**
    4.08
- **Browser and version:**
  - 98.0.1108.43
- **Operating system and version:**
  - Windows 11 (21H2)
- **Filters you use in AdGuard:**
  - Kees1958 and LennyFox bookmarks
- **Stealth Mode settings:**
  - disabled
- **Other browser extensions:**
  - none

### Expected Behavior
Dutch text should be: "Alle eigen filter regels verwijderen voor deze webpagina"

### Actual Behavior

Dutch text is: "Alle gebruikersregels herstellen voor deze webpagina"

### Screenshots
![image](https://user-images.githubusercontent.com/30925856/153421968-3db6d05f-1bf9-46bc-9111-ce56ced62b4d.png)

**Comment by Kees1958** [comment]
Yes, the explanation uses the translation for "user rules" (which is "gebruikersregels". Problem is that in the menu User rules, is called Own filters, which is obvious different. So you either have to change the menu option User rules to "Gebruikersregels" or you have to change the reference to User rules in text to Eigen filters.

Secondly I am pretty sure that "herstellen" means repair, while the actual action is to delete the User rules for that website.

**Comment by el-termikael** [comment]
I've changed the menu option to "Gebruikersregels", hopefully, the Dutch localization will be updated in the next version of AdGuard Browser Assistant.

As for "herstellen", I guess the translation is alright — the source string is `Reset all user rules for this page`, it's to reset, not to delete.

**Comment by Kees1958** [comment]
@el-termikael 

Helen, 

Try this sequence of actions:
1. Add some rules for website xyz.com
2. Give AG time to save the User Rules
3. The baseline is now the saved User Rules
4.  Add a new user rule by using the add rule wizard of AG
5.  Reset the newly created rules

All rules are removed, not just the last created rule, so the action is not reset, but remove

**Comment by el-termikael** [comment]
@Kees1958 Right you are. We'll change this string to `Remove all user rules for this page`
